save-frontend/src/main/kotlin/com/saveourtool/save/frontend/App.kt

```diff
@@ -112,17 +112,10 @@ class App : ComponentWithScope<PropsWithChildren, AppState>() {
             requestModalHandler {
                 userInfo = state.userInfo
 
-                withRouter<Props> { location, _ ->
```

Comment:
here was a bug with redirection (after some of updates such navigation does not work). And actually that was an incorrect  one from the very beginning